src/main/java/Hubert.java

```diff
@@ -0,0 +1,83 @@
+import java.util.Scanner;
+
+public class Hubert {
+
+    private static int markIndex(String line, int num) {
+        String markIndexChar = line.substring(num, line.length());
+        return Integer.parseInt(markIndexChar) - 1;
+    }
+
+    private static Task addTask(String line) {
```

Comment:
I think use line to indicate the input may be a bit confusing, I suppose use commandLine or userInput is better